ui/app/components/toolbar-download-button.js

```diff
@@ -0,0 +1,5 @@
+import DownloadButton from './download-button';
```

Comment:
I ended up adding this to add the icon to the `download-button` component, but if it's possible to tell it to use the `ToolbarButton` template then it may not be necessary

Reply:
You _might_ be able to use `classNames=''` in the template, but I'm not sure